src/site/stages/build/plugins/check-broken-links.js

```diff
@@ -63,7 +63,7 @@ function checkBrokenLinks() {
       }, {});
 
     brokenLinkChecker(filteredFiles);
-    removeLazySrcAttribute(filteredFiles);
+    removeLazySrcAttribute(files);
```

Comment:
This was also a bug I didn't notice because the only lazy loaded images are on the homepage and now there's a drupal homepage.